public/doc/swagger-2-v0.0.1.yaml

```diff
@@ -1389,6 +1389,11 @@ definitions:
         type: string
         format: date-time
         readOnly: true
+      taggings:
+        type: array
+        items:
+          "$ref": "#/definitions/Tagging"
```

Comment:
@bdunne hm, the specs require this to be a class. Should I create a class for this, just and instantiate it in https://github.com/ManageIQ/topological_inventory-core/commit/c5a91019ee50f81cca7ec681fc176cd375a9d0d0 ?

Or what is the right way? I assume we want to have this type check, that it can't be just array of anything.

Reply:
You can add it to the exception list for now https://github.com/ManageIQ/topological_inventory-api/blob/master/spec/swagger_spec.rb#L105
I'll probably have to do something different as we accumulate more non-model definitions.